extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/conf/CASGuacamoleProperties.java

```diff
@@ -68,5 +69,30 @@ private CASGuacamoleProperties() {}
         public String getName() { return "cas-clearpass-key"; }
 
     };
+  
+    /**
+     * The attribute used for group membership
+     * example:  memberOf  (case sensitive)
+     */
```

Comment:
This reads unclearly to me. At first glance, it looks like "(case sensitive)" is part of the example attribute value. JavaDoc convention also dictates that periods must be used.

Perhaps something more like:

```java
/**
 * The name of the attribute used for group membership, such as "memberOf".
 * This attribute is case sensitive.
 */